extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -335,6 +345,20 @@ public boolean isRunning() {
         }
     }
 
+    @Override
+    public List<Trigger> getScheduledJobs() {
+        return new ArrayList<>(scheduledTasks.values());
```

Comment:
The result is not immutable. So something like `return List.copyOf(scheduledTasks.values())`?